modules/formulize/templates/admin/ui-accordion.html

```diff
@@ -35,7 +35,7 @@
         <h3><a href="#"><span class="accordion-name"><{$section.name}></span></a></h3>
 		<div class="accordion-content content">
 			<{if $section.header}>
-			<div style="position: absolute; top: 10px; right: 10px;">
+			<div style="position: absolute; max-width: 50%; top: 10px; right: 10px;">

```

Comment:
Instead of having each of these side-by-side covering half of the area, I'd prefer to have them full-width and the toolbar above the description. Can you please change it? Also, move the "Add a Form" link from the left side into the group of links in the toolbar.